runtime/dart_isolate.cc

```diff
@@ -697,6 +697,8 @@ Dart_Isolate DartIsolate::DartCreateAndStartServiceIsolate(
                                 nullptr, nullptr, nullptr, nullptr);
 
   flags->load_vmservice_library = true;
+  flags->null_safety =
```

Comment:
It would be a little more safe if we guard this code as
#if (FLUTTER_RUNTIME_MODE != FLUTTER_RUNTIME_MODE_DEBUG)
  vm_data->GetIsolateSnapshot()->....
#endif

The reason being I feel the VM may hit an assert if a null value is passed for the script_uri.
Once we fix the VM to do the right thing we can adjust this code.

Reply:
Done. I added a TODO for the fix. PTAL.